src/app/logging.ts

```diff
@@ -1,7 +1,7 @@
 import { Category, CategoryLogger, CategoryServiceFactory, CategoryDefaultConfiguration, LogLevel } from 'typescript-logging';
 
 // TODO typescript-logging will support a console API to change this at runtime in the future, for now we'll use localStorage
-const storedLogLevel = localStorage.getItem('logLevel');
+const storedLogLevel = typeof localStorage !== 'undefined' && localStorage.getItem('logLevel');
```

Comment:
@gashcrumb are you ok with this change and `typescript-logging` version bump (#317)?